### PR TITLE
2BrightSparks.SyncBackProX64 version 10.2.68.0

### PR DESCRIPTION
**Pull request by 2BrightSparks**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/83153)

### CODE REVIEW

**Comment by 2BrightSparks** [comment]
@microsoft-github-policy-service agree [company="2BrightSparks Pte Ltd"]

**Comment by 2BrightSparks** [comment]
@microsoft-github-policy-service agree company="2BrightSparks Pte Ltd"

File: manifests/2/2BrightSparks/SyncBackProX64/10.2.68.0/2BrightSparks.SyncBackProX64.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: 2BrightSparks.SyncBackProX64
```
Comment (ItzLevvie) [inline manifests/2/2BrightSparks/SyncBackProX64/10.2.68.0/2BrightSparks.SyncBackProX64.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: 2BrightSparks.SyncBackPro
```
You'll have to change the `SyncBackProX64` folder to `SyncBackPro` ...
... and update the file names as well.

You can do this by pressing the `.` (dot) key on your keyboard which will open up Visual Studio Online.

File: manifests/2/2BrightSparks/SyncBackProX64/10.2.68.0/2BrightSparks.SyncBackProX64.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: 2BrightSparks.SyncBackProX64
```
Comment (ItzLevvie) [inline manifests/2/2BrightSparks/SyncBackProX64/10.2.68.0/2BrightSparks.SyncBackProX64.installer.yaml:4]:
```suggestion
PackageIdentifier: 2BrightSparks.SyncBackPro
```
You'll have to change the `SyncBackProX64` folder to `SyncBackPro` ...
... and update the file names as well.

You can do this by pressing the `.` (dot) key on your keyboard which will open up Visual Studio Online.

File: manifests/2/2BrightSparks/SyncBackProX64/10.2.68.0/2BrightSparks.SyncBackProX64.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: 2BrightSparks.SyncBackProX64
+PackageVersion: 10.2.68.0
+PackageLocale: en-US
+Publisher: 2BrightSparks Pte Ltd
+PackageName: SyncBackPro x64
```
Comment (ItzLevvie) [inline manifests/2/2BrightSparks/SyncBackProX64/10.2.68.0/2BrightSparks.SyncBackProX64.locale.en-US.yaml:8]:
```suggestion
PackageName: SyncBackPro
```

File: manifests/2/2BrightSparks/SyncBackProX64/10.2.68.0/2BrightSparks.SyncBackProX64.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: 2BrightSparks.SyncBackProX64
+PackageVersion: 10.2.68.0
+PackageLocale: en-US
+Publisher: 2BrightSparks Pte Ltd
+PackageName: SyncBackPro x64
+License: Commercial
+Copyright: 2BrightSparks Pte Ltd
+ShortDescription: SyncBackPro x64 V10.2.68.0
```
Comment (ItzLevvie) [inline manifests/2/2BrightSparks/SyncBackProX64/10.2.68.0/2BrightSparks.SyncBackProX64.locale.en-US.yaml:11]:
```suggestion
ShortDescription: SyncBackPro backup software, the professional's tool to backup, synchronize, and restore data files.
```

File: manifests/2/2BrightSparks/SyncBackProX64/10.2.68.0/2BrightSparks.SyncBackProX64.yaml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: 2BrightSparks.SyncBackProX64
```
Comment (ItzLevvie) [inline manifests/2/2BrightSparks/SyncBackProX64/10.2.68.0/2BrightSparks.SyncBackProX64.yaml:4]:
```suggestion
PackageIdentifier: 2BrightSparks.SyncBackPro
```
You'll have to change the `SyncBackProX64` folder to `SyncBackPro` ...
... and update the file names as well.

You can do this by pressing the `.` (dot) key on your keyboard which will open up Visual Studio Online.

File: manifests/2/2BrightSparks/SyncBackProX64/10.2.68.0/2BrightSparks.SyncBackProX64.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: 2BrightSparks.SyncBackProX64
+PackageVersion: 10.2.68.0
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://www.2brightsparks.com/assets/software/SyncBack/SyncBackPro64_Setup.10.2.68.0.exe
+  InstallerSha256: 930E9FC02C61703A26A00679CF48D77F7CD1A3FD718B4EECAD16EEB588E13996
```
Comment (ItzLevvie) [inline manifests/2/2BrightSparks/SyncBackProX64/10.2.68.0/2BrightSparks.SyncBackProX64.installer.yaml:10]:
```suggestion
  InstallerSha256: 930E9FC02C61703A26A00679CF48D77F7CD1A3FD718B4EECAD16EEB588E13996
  ProductCode: SyncBackPro64_is1_is1
  AppsAndFeaturesEntries:
  - ProductCode: SyncBackPro64_is1_is1
    DisplayName: SyncBackPro x64
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Pull request closed without merging**